### DISCUSSION TITLE
How fast is pixart-alpha's inference?

### DISCUSSION THREAD

**Discussion by Njasa2k** [PixArt-alpha/PixArt-alpha#10] [Q&A] [answered]
How fast is pixart-alpha's inference?

**Comment by lawrence-cj** [comment]
Working with the Diffusers team, the final inference speed will release once ready.

**Comment by yujincheng08** [comment, marked as answer]
According to the huggingface's team, they can generate a 1024x1024 image with 20 steps of DPM-solver in around 1.5s.
